static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-one-million-campaign-no-control.js

```diff
@@ -8,7 +8,7 @@ import {
     getLocalCurrencySymbol,
 } from 'lib/geolocation';
 
-const abTestName = 'AcquisitionsEpicOneMillionCampaign';
+const abTestName = 'AcquisitionsEpicOneMillionCampaignNoControl';
```

Comment:
because of `useLocalViewLog`, changing this name will re-display this epic even for people with more than 4 views. given that it's campaign week, maybe this is OK - but worth being aware of in case anyone asks